libvast/src/system/exporter.cpp

```diff
@@ -232,15 +237,15 @@ exporter(exporter_actor::stateful_pointer<exporter_state> self, expression expr,
         : query_context::priority::normal;
   VAST_DEBUG("spawned exporter with {} pipelines", pipelines.size());
   self->state.pipeline = pipeline_executor{std::move(pipelines)};
```

Comment:
Doesn't make sense to get results in batches in this case.
```suggestion
  self->state.pipeline = pipeline_executor{std::move(pipelines)};
  if (self->state.pipeline.is_blocking())
    self->state.query_context.taste = std::numeric_limits<uint32_t>::max();
```

Reply:
Nice catch!